src/serac/physics/base_physics.cpp

```diff
@@ -45,10 +47,14 @@ void BasePhysics::setTime(const double time) { time_ = time; }
 
 double BasePhysics::time() const { return time_; }
 
+double BasePhysics::adjointTime() const { return adjoint_time_; }
```

Comment:
This is the time during the adjoint stepping?

Reply:
yes - I thought it was better to have this be a separate field vs. the forward pass, but I am open to suggestions.

Reply:
have them be different implies they can sometimes go out of sync?  What is the situation where the adjoint and time would return different values?